{"name": "Unsuccessfully login with email: simeon.hhl.qa@gmail.co and password: Test#1234", "status": "passed", "steps": [{"name": "Check for error", "status": "passed", "start": 1732419425793, "stop": 1732419427813}, {"name": "Check the error text", "status": "passed", "start": 1732419427814, "stop": 1732419427833}], "parameters": [{"name": "email", "value": "'simeon.hhl.qa@gmail.co'"}, {"name": "password", "value": "'Test#1234'"}, {"name": "error", "value": "'There was a problem logging you in. Please try again.'"}, {"name": "driver", "value": "'edge'"}], "start": 1732419424944, "stop": 1732419427833, "uuid": "7b5ff133-71d4-456a-a1c4-fd8d9f85d536", "historyId": "2ffef19ed6000db1495424120c5ac1a1", "testCaseId": "c2811b1721d8231bbb0fb40778ae099c", "fullName": "Tests.test_Login.TestLogin#test_2", "labels": [{"name": "feature", "value": "Login"}, {"name": "severity", "value": "normal"}, {"name": "tag", "value": "flaky(reruns=3, reruns_delay=1)"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "fv-az801-993"}, {"name": "thread", "value": "1893-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}